CHANGELOG.md

```diff
@@ -3,7 +3,42 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.51.0
+
+This release contains improvements to monitoring and an assortment of fixes and improvements.
+
+### Monitoring
+
+- Surface unauthorized response count from OPA API authz handler ([#3378](https://github.com/open-policy-agent/opa/issues/3378)) authored by @ashutosh-narkar
+- plugins: Surface more decision log errors via status API ([#5637](https://github.com/open-policy-agent/opa/issues/5637)) authored by @ashutosh-narkar
+- plugins/rest: Include truncated HTTP response in logs ([#2961](https://github.com/open-policy-agent/opa/issues/2961)) authored by @ashutosh-narkar reported by @gshively11
+
+### Topdown and Rego
+
+- Wasm: Add native support for `object.union_n` built-in function (authored by @Azanul)
+
+### Fixes
+
+- ast: Relaxing strict-mode check for unused args in else-branching functions. ([#5758](https://github.com/open-policy-agent/opa/issues/5758)) authored by @johanfylling reported by @ethanjli
+- ast: Properly set the reported location of unused variables in strict-mode errors. ([#5662](https://github.com/open-policy-agent/opa/issues/5662)) authored by @boranx
+- fmt: report wrong arity for built-in functions. ([#5646](https://github.com/open-policy-agent/opa/issues/5646)) authored by @Trolloldem
+- topdown: http.send(): Ensuring intra-query caching consistency. ([#5736](https://github.com/open-policy-agent/opa/issues/5736)) authored by @johanfylling
+
+### Docs, Website, and Ecosystem
+
+- Fix typo in documentation (authored by @eternaltyro)
+- Update TLS authentication docs (authored by @charlieegan3)
+- Clarification in docs about checksums of Windows executables (authored by @Ronnie-personal)
+- docs: Small fix to context placement in integration (authored by @craigpastro)
+- docs/website: Fix floating navbar anchor issue ([5774](https://github.com/open-policy-agent/opa/issues/5774)) authored by @charlieegan3 reported by @kristiansvalland
+- Ecosystem:
+  - Add Quali Torque (authored by @shirabendor-quali)
+  
+### Miscellaneous
+
+- Update -debug images to use Chainguard images ([5544](https://github.com/open-policy-agent/opa/issues/5544)) (authored by @charlieegan3)
+- Various third-party dependencies were updated.
+- Performance improvements to decision logging
```

Comment:
This seems like something I'd want to read more about, so a link would be good! Also, I'd move this to the Fixes section unless we'll have one for performance,

Reply:
It's not much, just removed some superfluous work in that code path... we don't have hard numbers either... so we might as well drop it 🤷

Reply:
This is one of those PR:s with no attached issue; but perhaps @srenatus could come up with a short blurb describing the fix?

Reply:
"remove superfluous json encoding roundtrip and double work in AST conversion of to-be-logged events" ?